packages/core/ui/FileSelector.tsx

```diff
@@ -87,16 +95,115 @@ const FileLocationEditor = observer(
 const UrlChooser = (props: {
   location?: FileLocation
   setLocation: Function
+  internetAccounts?: Account[]
 }) => {
-  const { location, setLocation } = props
+  const { location, setLocation, internetAccounts } = props
+  const [currentUrl, setCurrentUrl] = useState('')
+  const [currentInternetAccount, setCurrentInternetAccount] = useState(
+    'autoDetect',
+  )
+
+  const autoDetectInternetAccount = (urlInput: string) => {
+    let detectedId = ''
+    try {
+      new URL(urlInput)
+    } catch (error) {
+      // skip
+      return detectedId
+    }
+
+    internetAccounts?.forEach(account => {
+      if (account.handlesLocation({ uri: urlInput })) {
+        detectedId = account.accountConfig.internetAccountId
```

Comment:
I think this should return the account name to ensure that there is consistency with how the configured internet accounts are displayed below it.

![image](https://user-images.githubusercontent.com/83305007/126798871-fbb35981-3be5-4245-9f4b-03973f84eed6.png)